valtio 1.5.0 in Next.js

After `yarn create next-app . && yarn add valtio`, 

when valtio is imported to ~/pages/index.js with `import { proxy, useSnapshot } from 'valtio'`, 

there is an error: 
`error - Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import '~/node_modules/use-sync-external-store/shim' is not supported resolving ES modules imported from ~/node_modules/valtio/esm/index.mjs Did you mean to import ~/node_modules/use-sync-external-store/shim/index.js?`

Any idea how to fix it?

i have the same error & `npm i pmndrs/valtio#fix/build/path-for-cjs` isn't working even tho i [do it every time as i have this answer bookmarked](https://stackoverflow.com/a/50734317/6141587).

also, tried setting it in `package.json` like:

```
"valtio": "github:pmndrs/valtio#fix/build/path-for-cjs"
```

& then `npm i` but still ain't working. i get an error:

```bash
npm ERR! The git reference could not be found
npm ERR! command git --no-replace-objects checkout fix/build/path-for-cjs
npm ERR! error: pathspec 'fix/build/path-for-cjs' did not match any file(s) known to git
```

never had `/` in branch name but is there any other way to install it?

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -69,6 +69,8 @@ function createESMConfig(input, output) {
         __DEV__: '(import.meta.env&&import.meta.env.MODE)!=="production"',
         'use-sync-external-store/shim/with-selector':
           'use-sync-external-store/shim/with-selector.js',
+        // a workround for #410
+        'use-sync-external-store/shim': 'use-sync-external-store/shim/index.js',
         preventAssignment: true,
       }),
       getEsbuild('node12'),

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -67,10 +67,9 @@ function createESMConfig(input, output) {
       resolve({ extensions }),
       replace({
         __DEV__: '(import.meta.env&&import.meta.env.MODE)!=="production"',
-        'use-sync-external-store/shim/with-selector':
-          'use-sync-external-store/shim/with-selector.js',
         // a workround for #410
         'use-sync-external-store/shim': 'use-sync-external-store/shim/index.js',
+        delimiters: ['\\b', '\\b(?!(\\.|/))'],
         preventAssignment: true,
       }),
       getEsbuild('node12'),

